invenio_circulation/utils.py

```diff
@@ -60,7 +60,7 @@ def is_item_available(item_pid):
     )
 
 
-def can_be_requested(item_pid=None, document_pid=None):
+def can_be_requested(loan):
     """Return if record can be requested."""
```

Comment:
I would change the comment then: I would put something more clear, like:
`
"""Should return True if document or item can be requested."""
`
because with record I understand `loan`.

Reply:
We have changed the comment.